libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigation.java

```diff
@@ -71,11 +70,10 @@ public class MapboxNavigation implements ServiceConnection {
    * </p>
    *
    * @param context     required in order to create and bind the navigation service
-   * @param accessToken a valid Mapbox access token
    * @since 0.5.0
    */
-  public MapboxNavigation(@NonNull Context context, @NonNull String accessToken) {
-    this(context, accessToken, MapboxNavigationOptions.builder().build());
+  public MapboxNavigation(@NonNull Context context) {
+    this(context, MapboxNavigationOptions.builder().build());
```

Comment:
Removing the access token is in line with what we want to do in general since MapLibre aims to be vendor-neutral.

Reply:
Code-wise, is it a breaking change here?

Reply:
OK then I would say let's bump the major version